src/global/postburn/postburn.cc

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Alex Jentsch, Jihee Kim, Brian Page
+//
+
+
+#include <Evaluator/DD4hepUnits.h>
+#include <JANA/JApplication.h>
+#include <vector>
+
+#include "algorithms/postburn/PostBurnConfig.h"
+#include "extensions/jana/JOmniFactoryGeneratorT.h"
+#include "factories/postburn/PostBurnMCParticles_factory.h"
+
+
+
+extern "C" {
+void InitPlugin(JApplication *app) {
+    InitJANAPlugin(app);
+    using namespace eicrecon;
+
+
+    PostBurnConfig postburn_config;
+
+
+    //Full correction for MCParticles --> MCParticlesHeadOnFrame
+    postburn_config.pidAssumePionMass = false;
+    postburn_config.crossingAngle    = -0.025 * dd4hep::rad;
+    postburn_config.pidPurity        = 0.51; //dummy value for MC truth information
+    postburn_config.correctBeamFX    = true;
+    postburn_config.pidUseMCTruth    = true;
+
+
+    //Need to read-in MCParticles
+
+    app->Add(new JOmniFactoryGeneratorT<PostBurnMCParticles_factory>(
+            "MCParticlesHeadOnFrameNoBeamFX",
+            {
+                "MCParticles"
+            },
+            {
+                "MCParticlesHeadOnFrameNoBeamFX"
+            },
+            postburn_config,
```

Comment:
Please define inline.
```suggestion
            {
              .crossingAngle = -0.025 * dd4hep::rad;
            },
```

Reply:
Rest of the code uses inline structure intializations. Uniformity allows easier understanding of the codebase for new users.